Project.toml

```diff
@@ -7,18 +7,19 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Mimi = "e4e893b0-ee5e-52ea-8111-44b3bdec128c"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-CSVFiles = "5d742f6a-9f54-50ce-8119-2520741973ca"
-DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
-
 [compat]
 julia = "1"
 Mimi = "0.9.3"
```

Comment:
@ckingdon95 I think we'll tag a new release of `Mimi` first and change this to `Mimi = "0.9.4" to handle the fix to the MarginalModel bug in the simulation `run` function